microsoftgraph/microsoft-graph-explorer-v2
# Feature/permissions consent

## Overview

- Changes incremental consent implementation to per scope consent
- Removes message directing users to EdgeDev
- Allows login via Redirect / Popup

Fixes #256 


### Demo

![image](https://user-images.githubusercontent.com/53429535/67786952-c9ce2900-fa80-11e9-830c-31d012f90190.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Log in
* Select a query to run
* Open the permissions tab to view the permissions required for the query
* Click the consent button against the permission you need and get the consent required interaction
* Notice that the permission now reads consented
